src/platform/plugins/shared/discover/public/application/main/state_management/redux/internal_state.ts

```diff
@@ -193,11 +193,16 @@ export interface InternalStateThunkDependencies {
   urlStateStorage: IKbnUrlStateStorage;
 }
 
+const IS_JEST_ENVIRONMENT = typeof jest !== 'undefined';
+
 export const createInternalStateStore = (options: InternalStateThunkDependencies) => {
   const store = configureStore({
     reducer: internalStateSlice.reducer,
     middleware: (getDefaultMiddleware) =>
-      getDefaultMiddleware({ thunk: { extraArgument: options } }),
+      getDefaultMiddleware({
+        thunk: { extraArgument: options },
+        serializableCheck: !IS_JEST_ENVIRONMENT,
```

Comment:
I disabled the RTK serialization check for Jest tests because a lot of our tests mock state with unserializable values (e.g. data views) which was creating a bunch of noise when debugging test failures. I'm open to reverting this though if someone feels strongly that it's a bad idea.